File: react-native-otvplayer/otvplayer-0.67.4/distTemplate/ios/react-otvplayer/ReactOtvplayer.m
// Copyright (c) 2020--2023 Nagravision SA. All rights reserved.
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(ReactOtvplayer, NSObject)

RCT_EXTERN_METHOD(multiply:(float)a withB:(float)b
                 withResolver:(RCTPromiseResolveBlock)resolve
                 withRejecter:(RCTPromiseRejectBlock)reject)

@end

File: react-native-otvplayer/otvplayer-0.72.4/distTemplate/ios/react-otvplayer/ReactOtvplayer.swift
// Copyright (c) 2020--2023 Nagravision SA. All rights reserved.
@objc(ReactOtvplayer)
class ReactOtvplayer: NSObject {

    @objc(multiply:withB:withResolver:withRejecter:)
    func multiply(a: Float, b: Float, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
        resolve(a*b)
    }
}
